app/graphql/albums/resolvers.js

```diff
@@ -0,0 +1,20 @@
+const config = require('../../../config'),
+  { url } = config.common.apiAlbums,
+  { getAlbumSources } = require('../../services/albums');
+
+const getAlbum = (_, params) =>
+  getAlbumSources(`${url}albums/${params.id}`).then(album => ({
+    id: album.id,
+    title: album.title,
+    artist: album.userId
```

Comment:
you can save yourself this serialization with the resolvers of the type Album! It is your choice though, either way works fine

Reply:
ok!